compiler/driver/src/output.rs

```diff
@@ -1,5 +1,10 @@
+use std::fs::create_dir_all;
 use std::path::PathBuf;
 
+use anyhow::*;
+
+use thiserror::private::PathAsDisplay;
```

Comment:
Is this a private API? You should be able to just do `format!("{}", path.display())` if you need to format a `Path` or `PathBuf` value.

Reply:
IntelliJ Rust suggested it, but I can use the other API.